src/Controllers/DataServiceController.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using NetTopologySuite.Geometries;
+using System.Text.Json;
+
+namespace EWS;
+
+[ApiController]
+[Route("[controller]")]
+public class DataServiceController : ControllerBase
+{
+    /// <summary>The well-known layer names for querying data.</summary>
+    private const string GemeindeLayer = "ch.so.agi.gemeindegrenzen.data";
+    private const string GrundstueckLayer = "ch.so.agi.av.grundstuecke.rechtskraeftig.data";
+
+    private readonly HttpClient client;
+    private readonly ILogger<DataServiceController> logger;
+
+    public DataServiceController(HttpClient client, ILogger<DataServiceController> logger)
+    {
+        this.client = client;
+        this.logger = logger;
+
+        client.BaseAddress = new Uri("https://geo.so.ch/api/data/v1/");
+    }
+
+    /// <summary>
+    /// Retrieves the Gemeinde and Grundbuchnummern for the given points from the Data Service API.
+    /// </summary>
+    /// <param name="points">A list of <see cref="Point"/> to get the information for.</param>
+    /// <returns>A <see cref="DataServiceResponse"/> containing the Gemeinde and Grundbuchnummern information.</returns>
+    [HttpGet]
+    public async Task<ActionResult<DataServiceResponse>> GetAsync([FromQuery] List<Point> points)
+    {
+        try
+        {
+            List<string?> gemeinden = new(), grundbuchNummern = new();
+            foreach (var point in points)
+            {
+                var boundingBox = $"{point.EnvelopeInternal.MaxX},{point.EnvelopeInternal.MaxY},{point.EnvelopeInternal.MaxX},{point.EnvelopeInternal.MaxY}";
```

Comment:
Q: Ist es sinnvoll für einen einzelnen Punkt jeweils eine Bounding Box zu berechnen? Und muss man den Api Aufruf dann für jeden Punkt einzeln machen und kann nicht eine Boundingbox berechnen die alle Punkte enthält?

Reply:
@MiraGeowerkstatt Ja ich denke schon. Denn je nach dem kann die Bounding Box dann in einer anderen Gemeinde oder einem anderen Grundstück liegen als die Bohrung selber.

Reply:
Q: Ist die bounding box anders wenn man das hier macht?
$"{point.x},{point.y},{point.x},{point.x}"